partcad/src/partcad/shape.py

```
@@ -394,9 +396,7 @@
 
             def do_render_png():
                 nonlocal project, svg_path, width, height, filepath
-                plugins.export_png.export(
-                    project, svg_path, width, height, filepath
-                )
+                plugins.export_png.export(project, svg_path, width, height, filepath)
```

Ruff (0.8.2)

`plugins` may be undefined, or defined from star imports

(F405)